src/main/java/menelaus/view/BoardView.java

```diff
@@ -13,53 +13,26 @@
  */
 public class BoardView extends JPanel {
 	
-
+	public Dimension getPreferredSize() {
+		return new Dimension(700, 700);
+	};
+	
     /**
      * Core board.
      */
     Board board;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://www.codacy.com/app/dmvouldjeff/menelaus/file/2444527322/issues/source?bid=3228196&fileBranchId=3246061#l23)